src/README.md

```diff
@@ -1,4 +1,4 @@
-# Parsec
+![Parsec Logo](parsec.png)
```

Comment:
It's jsut to print the logo on the main page instead of the "Parsec" title.

![image](https://user-images.githubusercontent.com/29229160/111307031-d5f45f80-8650-11eb-9ea9-7e7190fa750e.png)

Looks better 😎